azure-pipelines.yml

```diff
@@ -45,8 +45,7 @@ variables:
 jobs:
 - template: build-pipeline.yml@templates
   parameters:
-    skip_tests: true
-    has_ui_tests: false
+    has_unit_tests: false
```

Comment:
is `has_unit_tests` better than `skip_tests`?

Reply:
skip_tests will be deprecated